apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Plugin Name: Full Site Editing
+ */
+
+class WPCOM_Full_Site_Editing {
+	function __construct() {
+		$this->register_script_and_style();
+
+		add_action( 'init', [ $this, 'register_blocks' ], 100 );
```

Comment:
I love this, but if we're going to send this to Jetpack then we need to make sure we don't use the array shorthand syntax

```php
add_action( 'init', array( $this, 'register_blocks' ), 100 );
```

Reply:
Dang!
I was sure JP was on PHP 7 nowadays, but thinking about it, I guess I got confused with WPCOM.

Reply:
Jetpack is soon able to bump PHP requirement and PHP 5.4 might already support this? Are you targeting  7.3 or 7.4 release? (@kraftbj and @jeherve prolly know more)

Reply:
tl;dr: ff you are aiming to launch in Jetpack 7.3 (or even 7.4), it's best to stick to PHP 5.2 compatibility. 

*****

Jetpack aims to follow the same requirements as WordPress core, and supports Core `n-1`. For example, right now Core is 5.1, so Jetpack's minimum supported version is 5.0.

WordPress 5.2, scheduled to be released in a couple of weeks, will up the PHP requirements to PHP 5.6.

As a result, we'll be able to start using PHP 5.6 syntax as soon as WordPress 5.3 ships, since at that point Jetpack will support WordPress 5.2.

Reply:
Core is theoretically [moving to PHP 5.6](https://make.wordpress.org/core/2018/12/08/updating-the-minimum-php-version/) this month and Jetpack follows suit so hopefully this will be moot in the not-too-distant future

Reply:
We are not targeting any particular release at this stage, this is mainly for internal development and milestones. Also at this point it's not clear if and how we'll incorporate this into Jetpack.